module/actors/sheets/BWCharacterSheet.ts

```diff
@@ -84,7 +84,7 @@ export class BWCharacterSheet extends BWActorSheet<CharacterSheetData, BWCharact
                     const s = i as SkillDataRoot;
                     if (s.data.learning) {
                         learning.push(s);
-                    } else if (s.data.training && ["martial", "training"].includes(s.data.skilltype)) {
+                    } else if (s.data.training) {
```

Comment:
This is a bit inaccurate to what the correct fix would be.

Though this might not be clear in the issue, the primary case this does not catch is the Formation Fighting skill, which is a military training skill.

A more extensive rework would be needed to get the correct behaviour, but for now, just adding `"military"` to the specially considered skill types.